assets/js/teams.js

```diff
@@ -18,20 +18,20 @@ jQuery(function () {
     {
       data: 'attributes.products',
       createdCell: function (td, _cellData, rowData) {
-        for (const key in rowData.attributes.products) {
-          var pList = ''
-          rowData.attributes.products[key].forEach(product => {
-            pList += `<li><a href="/products/${product.attributes.slug}" data-test="product-${product.id}">${product.attributes.name}</a></li>`
+        const products = rowData.attributes.products.data
+          .map(product => {
+            return `<li><a href="/products/${product.attributes.slug}" data-test="product-${product.id}">${product.attributes.name}</a></li>`
```

Comment:
Just for learning, with arrow functions you can rewrite this as below, if you're just doing a return you can usually skip it and the brackets:

```
(product => `<li><a href="/products/${product.attributes.slug}" data-test="product-${product.id}">${product.attributes.name}</a></li>`)
```

Reply:
Modified as suggested